src/state/dialogs/index.tsx

```diff
@@ -53,7 +54,10 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
   >('auto')
 
   const closeAllDialogs = React.useCallback(() => {
-    activeDialogs.current.forEach(dialog => dialog.current.close())
+    openDialogs.current.forEach(id => {
+      const dialog = activeDialogs.current.get(id)
+      if (dialog) dialog.current.close()
+    })
```

Comment:
When this was originally written we didn't have a reference to ONLY the open dialogs. Now we do, so can close only those. `id` is the same. Test using the Storybook components.